doc/install/install-kubernetes.md

```diff
@@ -0,0 +1,120 @@
+```
+
+### Install [OLM](https://olm.operatorframework.io/)
+
+```bash
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.23.1/install.sh | bash -s v0.23.1
+```
+
+### Install Istio as a Gateway API provider
+
+> [!NOTE]  
+> There are several ways to install Istio (via `istioctl`, Helm chart or Operator) - this is just an example for starting from a bare Kubernetes cluster.
+
+
+```bash
+curl -sSL https://istio.io/downloadIstio | ISTIO_VERSION=1.20.7 sh -
+./istio-1.20.7/bin/istioctl install --set profile=minimal -y
+./istio-1.20.7/bin/istioctl operator init
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/kuadrant-operator/main/config/dependencies/istio/istio-operator.yaml
+```
+
+### Install Kuadrant
+
+```bash
+kubectl create -f https://operatorhub.io/install/kuadrant-operator.yaml
+```
+
+### Request a Kuadrant instance
+
+```bash
+kubectl create namespace kuadrant-system
+kubectl -n kuadrant-system apply -f - <<EOF
+apiVersion: kuadrant.io/v1beta1
+kind: Kuadrant
+metadata:
+  name: kuadrant
+spec: {}
+EOF
+```
+
+Kuadrant should now install. You can check the operator's install status with:
+
+```bash
+kubectl wait --for=jsonpath='{.status.state}'=AtLatestKnown subscription/my-kuadrant-operator -n operators --timeout=600s
+```
+
+Kuadrant is now ready to use.
+
+
+### (Optional) `DNSPolicy` setup
+
+If you plan to use `DNSPolicy`, you will need an AWS Account with access to Route 53 (more providers coming soon), and a hosted zone.
+
+Export the following environment variables for setup:
+
+```bash
+export AWS_ACCESS_KEY_ID=xxxxxxx # Key ID from AWS with Route 53 access
+export AWS_SECRET_ACCESS_KEY=xxxxxxx # Access key from AWS with Route 53 access
+```
+
+Create an AWS credentials secret:
+
+```bash
+kubectl -n kuadrant-system create secret generic aws-credentials \
+  --type=kuadrant.io/aws \
+  --from-literal=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+  --from-literal=AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+```
+
+### (Optional) Multi-cluster `RateLimitPolicy`
+If you want `RateLimitPolicy` to use shared, multicluster counters for Kuadrant's Limitador component, you'll need to configure Kuadrant with a Redis cluster URL. Create this secret:
+
+```bash
+# Replace this with an accessible Redis cluster URL
+export REDIS_URL=redis://user:xxxxxx@some-redis.com:6379
+kubectl -n kuadrant-system create secret generic redis-config \
+  --from-literal=URL=$REDIS_URL
+```
+
+You'll also need to update your earlier created `Kuadrant` instance:
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: kuadrant.io/v1beta1
+kind: Kuadrant
+metadata:
+  name: kuadrant
+  namespace: kuadrant-system
+spec:
+  limitador:
+    storage:
+      redis-cached:
+        configSecretRef:
+          name: redis-config 
+EOF
+```
+
+## Next Steps
+
+- [Secure, protect, and connect APIs with Kuadrant on OpenShift](../user-guides/secure-protect-connect-single-multi-cluster.md)
```

Comment:
I'm running through the follow on, and finding some things that may not make sense when following on from this guide rather than the local kind cluster guide. 
I'll add them as I find them, and make a recommendation how we can handle both approaches with the 1 follow on guide.

First thing is the cluster name when setting KUBECTL_CONTEXT is probably going to be different.
We can update the wording there to make it clear what's expected.

Then, the absence of a ClusterIssuer.
For now, I've done this:
```shell
kubectl apply -f - <<EOF
apiVersion: cert-manager.io/v1
kind: ClusterIssuer
metadata:
  name: kuadrant-operator-glbc-ca
spec:
  selfSigned: {}
EOF
```

Reply:
Crated a ManagedZone
```shell
export AWS_DNS_PUBLIC_ZONE_ID=<zone-id>

kubectl apply -f - <<EOF
apiVersion: kuadrant.io/v1alpha1
kind: ManagedZone
metadata:
  name: dev-mz
  namespace: kuadrant-system
spec:
  id: ${AWS_DNS_PUBLIC_ZONE_ID}
  domainName: ${KUADRANT_ZONE_ROOT_DOMAIN}
  description: "Dev Managed Zone"
  dnsProviderSecretRef:
    name: aws-credentials
EOF

```

Reply:
All good after that point.
Got to the end of the follow on guide at https://docs.kuadrant.io/0.8.0/kuadrant-operator/doc/user-guides/secure-protect-connect/

I used an external kubernetes cluster and redis instance in digitalocean rather than local kind clusters to catch any steps that only work with the local setup.